model/Security/Vocabularies/SsvcDecisionType.md

```diff
@@ -0,0 +1,23 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# SsvcDecisionType
+
+## Summary
+
+TODO
+
+## Description
+
+A SsvcDecisionType is TODO
+
+## Metadata
+
+- name: SsvcDecisionType
+
+## Entries
+
+- act: TODOdescription
+- attend: TODOdescription
+- track: TODOdescription
+- track*: TODOdescription - TODO: can entries have asterisks?
```

Comment:
track* - "*" will cause problems, needs to be updated to be text.

Reply:
This is the value per the specification unfortunately

Reply:
OK, should we use `trackStar`?  `trackPlus` ?

Reply:
I'd leave it as is if possible. If there's no way around changing it I'd recommend that we use `trackStar`

Reply:
Considering the variety of formats SPDX can be encoded into, my guess is that the asterisk will cause all sorts of unexpected problems. We should change it

Reply:
ok let's go with `trackStar`

Reply:
Renamed to `trackStar`